my.repos

```diff
@@ -34,7 +34,4 @@ repositories:
   symbolic_fact_generation:
     type: git
     url: https://github.com/DFKI-NI/symbolic_fact_generation.git
-    version: main
-  embedded-systems-bridge:
-    type: git
-    url: https://github.com/aiplan4eu/embedded-systems-bridge.git
+    version: main
```

Comment:
This file is now missing a newline at the end. Please see https://github.com/DFKI-NI/mobipick_labs#pre-commit-formatting-checks on how to automatically check and fix these formatting errors.